src/index.html

```diff
@@ -2,11 +2,284 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="home" id="home">
+    <header class="header">
+      <a href="#home" class="link_image">
+        <img src="images/logo.png" alt="logo"
+        class="image">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="is-active nav__item-link">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Smartphones</a>
+          </li>
```

Comment:
Recheck your formatting everywhere

https://mate-academy.github.io/style-guides/htmlcss.html
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="is-active nav__item-link">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__item-link">
              Samsung
            </a>
          </li>
```